Add Maven CLI usage example to Java generator documentation

Naveenkumarsuk opened the pull request:
This PR improves the documentation for the Java generator by adding an example that shows how to check the plugin version using the Maven CLI. This addition helps users quickly verify which version of the OpenAPI Generator is configured in their project.

Let me know if any changes are needed — happy to update.

TY

wing328 commented on docs/generators/java.md:
````diff
@@ -368,3 +368,10 @@ These options may be applied as additional-properties (cli) or configOptions (pl
+To check which version of the OpenAPI Generator Maven plugin is configured in your project, use:
+
+```bash
+mvn openapi-generator:version
````
this doc is for the java client generator

for maven plugin doc, please refer to https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator-maven-plugin/README.md

feel free to file a PR to update this readme to improve the maven plugin doc

Naveenkumarsuk commented:
Thanks for the guidance earlier. I’ve updated the PR to improve the Maven plugin documentation instead of the auto-generated Java generator doc.

I've added a usage example in modules/openapi-generator-maven-plugin/README.md showing how to generate a Java client using the Maven plugin.

Let me know if there’s anything I should refine further — happy to adjust

TY
